src/legend.ts

```diff
@@ -0,0 +1,237 @@
+export module LegendHTML {
+    function genSVGArrow(color: string, dashed: boolean, double: boolean, start: string, end: string, urlName: string, markerID: string, doubleMarkerID: string): SVGElement{
+        const svgURL = "http://www.w3.org/2000/svg"
```

Comment:
If you nest this constant declaration under the module `LegendHTML` instead of this function, you can also use it when you reference the URL in `appendTableElement()`.

Reply:
I nested the svgURL constant under the module LegendHTML to avoid redundant code.